engines/gob/resources.cpp

```diff
@@ -177,7 +177,7 @@ bool Resources::load(const Common::String &fileName) {
 	}
 
 	if (!hasTOTRes && !hasEXTRes)
-		return false;
+		return true;
```

Comment:
Looks suspicious

Reply:
Reverted to the old behavior for other games (it is needed for Adibou2, as some ancillary .TOT indeed to dot have any resource attached).